Makefile

```diff
@@ -235,4 +235,8 @@ test-miri:
 	$(cargo) +$(nightly) clean
 	MIRIFLAGS="-Zmiri-disable-isolation" $(cargo) +$(nightly) miri test
 
-.PHONY : build check build-release clippy install run-ledger run-gossip reset-ledger test test-debug fmt watch clean build-doc doc build-wasm-scripts-docker build-wasm-scripts clean-wasm-scripts dev-deps test-miri
+copy-wasm:
+	rm -rf .anoma/anoma-devchain.000000000000000/wasm/*.wasm .anoma/anoma-devchain.000000000000000/wasm/*.json
```

Comment:
I think we can remove this, it's been fixed in https://github.com/anoma/anoma/pull/936, which uses the root wasm directory directly in dev-mode.

Reply:
ah nice! ill remove it :)